src/error.rs

```diff
@@ -140,6 +140,12 @@ impl From<std::io::Error> for Error {
     }
 }
 
+impl From<std::cell::BorrowMutError> for Error {
```

Comment:
I think it would be better to just handle this error explicitly in the TailsFileReader

Reply:
Yeah, I was in doubt whether to do that as well. I'll move this then.

Reply:
Handled this explicitly.